paparazzi-gradle-plugin/src/test/projects/custom-fonts/build.gradle

```diff
@@ -17,6 +17,9 @@ android {
   kotlinOptions {
     jvmTarget = libs.versions.javaTarget.get()
   }
+
+  // For testing loading font resources from directories other than 'res/'
+  sourceSets.main.res.srcDirs += 'src/main/other_resources_dir'
```

Comment:
nit: this looks good but might be better in a separate test project. Not a blocker for this PR.